lib/process-services-cloud/src/lib/form/components/widgets/data-table/helpers/data-table-path-parser.helper.ts

```diff
@@ -80,8 +97,23 @@ export class DataTablePathParserHelper {
         return properties;
     }
 
+    getIndexReferencesFromProperty(property: string): number[] {
+        const match = this.indexReferencesRegEx.exec(property);
+
+        return match ? match[0].slice(1, -1).split('][').map(Number) : [];
```

Comment:
maybe to make it more clear, we can extract the `match[0].slice(1, -1).split('][').map(Number)` (if i understand it correctly)

to something like this?
```suggestion
private parseMatch(match: RegEx): number[] {
    const stringWithoutBrackets = match[0].slice(1, -1);
    const splitToStringArray = stringWithoutBrackets.split('][');
    const mapStringToNumberArray = stringArray.map(Number);

    return numberArray;
}
```
or even extract these operations to separate methods?